src/shared/services/UserService.ts

```diff
@@ -55,6 +55,9 @@ export class UserService {
 
     HttpService.client.logout();
 
+    // TODO: Remove this in a few releases when this cache has been deleted from most users' browsers
+    window.caches.delete("instance-cache");
```

Comment:
You can already remove this line from the main branch, it will take a few more months and a couple more 0.19.x releases until 1.0 gets published